COMET.Web.Common/Pages/Callback.razor

```diff
@@ -0,0 +1,23 @@
+﻿<!------------------------------------------------------------------------------
+// Copyright (c) 2023-2024 Starion Group S.A. 
+//
+//   Authors: Sam Gerené, Alex Vorobiev, Alexander van Delft, Jaime Bernar, Théate Antoine, João Rua
+//
+//   This file is part of COMET WEB Community Edition
+//   The COMET WEB Community Edition is the Starion Web Application implementation of ECSS-E-TM-10-25
+//   Annex A and Annex C.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+------------------------------------------------------------------------------->
+@page "/callback"
+Authentication in progress...
```

Comment:
please add some HTML to make this page look a bit nicer, potentially a loading spinner -> https://github.com/STARIONGROUP/SysML2.NET/blob/0602f9cb2c3875b5441b5ca7f41d5f50cb30071f/SysML2.NET.Viewer/wwwroot/index.html#L39